devtools/platform-descriptor-json/src/main/java/io/quarkus/platform/descriptor/loader/json/impl/QuarkusJsonPlatformDescriptorLoaderImpl.java

```diff
@@ -20,7 +21,7 @@ public QuarkusJsonPlatformDescriptor load(final QuarkusJsonPlatformDescriptorLoa
                     try {
                         ObjectMapper mapper = new ObjectMapper()
                                 .enable(JsonParser.Feature.ALLOW_COMMENTS)
-                                .enable(JsonParser.Feature.ALLOW_NUMERIC_LEADING_ZEROS)
```

Comment:
As the plan is to backport it, I would prefer we don't change this behavior and only do it in a follow-up PR for 1.8.

Reply:
yeah, I would leave that change out to reduce change. Not that i even know why we ever would have a file with leading zeros ;)

Reply:
Sure. FYI that property is deprecated for a long time now and should be removed in future Jackson versions

Reply:
I have no objection trying to remove it in 1.8.